foyer/tests/test_gaff_parameters.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from foyer import forcefields
+from foyer.exceptions import MissingParametersError
+
+
+@pytest.mark.skipif(
+    condition=not "load_GAFF" in dir(forcefields), reason="GAFF Plugin is not installed"
+)
+class TestGAFFParameters:
+    @pytest.fixture(scope="session")
+    def gaff(self):
+        return forcefields.load_GAFF()
+
+    def test_missing_group(self, gaff):
+        with pytest.raises(ValueError):
+            gaff.get_parameters("missing", key=[])
+
+    def test_non_string_keys(self, gaff):
+        with pytest.raises(TypeError):
+            gaff.get_parameters("atoms", key=1)
+
+    def test_bond_parameters_gaff(self, gaff):
+        bond_params = gaff.get_parameters("bonds", ["br", "ca"])
+        assert bond_params["length"] == 0.19079000000000002
```

Comment:
does this parameter go this far out in the decimals?

If not, it is probably better to use [np.isclose](https://numpy.org/doc/stable/reference/generated/numpy.isclose.html)

Reply:
I agree, `np.isclose` is probably better here. The bond length is `0.19079`.

@justinGilmer, for your reference, when I convert the the `parm.dat` files that are distributed with `AmberTools` to the `gaff.xml`, I do not do any rounding, to try to keep the numbers as faithful to the distributed values as possible. This is why the value has so many digits after the decimal point.

Reply:
What is what I was also assuming. It seemed like floating point precision, glad to confirm it!

Reply:
I agree. `np.allclose` is the way to go here.